pylibui/libui/box.py

```diff
@@ -35,33 +35,32 @@ def uiBoxAppend(box, child, stretchy):
     :return: None
     """
 
-    return clibui.uiBoxAppend(box, child, stretchy)
+    clibui.uiBoxAppend(box, child, stretchy)
 
 
-# - void uiBoxDelete(uiBox *b, uintmax_t index);
-def uiBoxDelete(*args):
+# - void uiBoxDelete(uiBox *b, int index);
```

Comment:
You don't need to change the function signature, of course. It's just for reference when implementing with ctypes..